### PR TITLE
EX-1498 - Style the sicherearbeit.at header

### PR DESCRIPTION
**Pull request by Bejor**
Bildschirmfoto 2021-09-20 um 17 37 21
Bildschirmfoto 2021-09-20 um 17 46 02

### CODE REVIEW

File: src/components/ex-navigation/ExNavigation.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
+.navigation {
+  width: 20% !important;
```
Comment (martinkastler) [inline src/components/ex-navigation/ExNavigation.scss:2]: There is one media query that does not yet show the burger, but the search is pushed to the next row. It is not a catastrophe, but maybe you can cover this case somehow.

image

File: src/components/ex-navigation/ExNavigation.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        @apply text-s bg-primary inline-block px-2 pt-1 pb-0 mr-8 text-white uppercase list-none transition-all duration-300 ease-in-out;
+
+        &:hover {
+          @apply bg-interactive-state-hover-button;
+        }
+      }
+    }
+  }
+
+  .ex-menu-burger {
```
Comment (martinkastler) [inline src/components/ex-navigation/ExNavigation.scss:59]: The burger menu item have no mouse-over effect. You could use the same background color change like the regular navigation. Should work.

image

**Review by martinkastler — CHANGES_REQUESTED** [review]
That AUVA banner at the top of the demo page just expands endlessly in with. I guess the layout will be set to a fixed with at some point? Otherwise we need to provide that header as SVG to stay sharp.

image

File: src/components/ex-navigation/ExNavigation.scss (hunk 2 above)
Comment (Bejor) [reply src/components/ex-navigation/ExNavigation.scss:63]: Added hover effect

File: src/components/ex-navigation/ExNavigation.scss (hunk 1 above)
Comment (Bejor) [reply src/components/ex-navigation/ExNavigation.scss:2]: fixed media query

**Comment by Bejor** [comment]
> That AUVA banner at the top of the demo page just expands endlessly in with. I guess the layout will be set to a fixed with at some point? Otherwise we need to provide that header as SVG to stay sharp.
> 
> image

Fixed max-width

File: src/components/ex-navigation/ExNavigation.scss (hunk 1 above)
Comment (martinkastler) [reply src/components/ex-navigation/ExNavigation.scss:2]: Menu items and search should be in a row:
image

File: src/components/ex-navigation/ExNavigation.scss (hunk 2 above)
Comment (martinkastler) [reply src/components/ex-navigation/ExNavigation.scss:63]: Now there is a double hover effect :D

image

Comment (Bejor) [reply src/components/ex-navigation/ExNavigation.scss:67]: My bad. I removed the wrong hover effect

**Review by martinkastler — CHANGES_REQUESTED** [review]
.navigation .ex-navigation__item does not require a padding-top.

image

Looks even without:

image

**Review by martinkastler — APPROVED** [review]

**Pull request merged**